plugins/e57/io/E57Reader.cpp

```diff
@@ -215,8 +248,22 @@ bool E57Reader::fillPoint(PointRef& point)
         auto dim = e57plugin::e57ToPdal(keyValue.first);
 
         if (dim != Dimension::Id::Unknown)
-            point.setField(dim,
-                           m_scan->rescale(dim, keyValue.second[m_currentIndex]));
+        {
+            auto val = keyValue.second[m_currentIndex];
+            if (dim != Dimension::Id::Classification)
```

Comment:
Why is Classification different? What happens to (x,y,z) and normal vectors? They should never be rescaled.